packages/flutter_test/lib/src/binding.dart

```diff
@@ -1785,24 +1785,38 @@ class _MockHttpHeaders extends HttpHeaders {
 class _AssetMocker {
   _AssetMocker();
 
-  static bool _mocked = false;
+  static Set<dynamic> _allowedKeys;
```

Comment:
Are these always strings?

Reply:
It should be. will you suggest using Set<String>?

Reply:
`Set<String>` would be preferable to `dynamic`.

Reply:
fixed